.github/workflows/test.yml

```diff
@@ -56,7 +56,7 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - name: Docker Build
-        run: docker build . --no-cache
+        run: docker build . --no-cache --build-arg IBC_GO_VERSION=v4.3.0
```

Comment:
the final version doesn't really matter here, this test just ensures the docker build succeeds. This image isn't used for anything.